scripts/db_migrator.py

```diff
@@ -636,6 +636,18 @@ def migrate_route_table(self):
             if 'protocol' not in route_attr:
                 self.appDB.set(self.appDB.APPL_DB, route_key, 'protocol', '')
 
+    def migrate_dns_nameserver(self):
```

Comment:
Will you create a new DBMigrator version? If no, I guess there will be overhead to warm reboot.

Reply:
Now I use the latest version: version_4_0_3

Reply:
Use version_4_0_4